backend/dataline/services/llm_flow/nodes.py

```diff
@@ -103,6 +107,42 @@ def run(cls, state: QueryGraphState) -> QueryGraphStateUpdate:
         # We return a list, because this will get added to the existing list
         return state_update(messages=output_messages, results=results)
 
+    @staticmethod
+    def fix_openai_multi_tool_use_bug(buggy_tool_call: ToolCall) -> list[ToolCall]:
+        """
```

Comment:
This docstring is a little confusing, could use some context at the beginning saying "This is the structure. I stared at it for a bit before realizing its a docstring lol.